packages/teams-js/src/public/interfaces.ts

```diff
@@ -267,81 +290,107 @@ export interface Context {
   tid?: string;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.app.theme} instead
+   *
    * The current UI theme.
    */
   theme?: string;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.page.isFullScreen} instead
+   *
    * Indication whether the tab is in full-screen mode.
    */
   isFullScreen?: boolean;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.team.type} instead
+   *
    * The type of the team.
    */
   teamType?: TeamType;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.sharePointSite.teamSiteUrl} instead
+   *
    * The root SharePoint site associated with the team.
    */
   teamSiteUrl?: string;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.sharePointSite.teamSiteDomain} instead
+   *
    * The domain of the root SharePoint site associated with the team.
    */
   teamSiteDomain?: string;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.sharePointSite.teamSitePath} instead
+   *
    * The relative path to the SharePoint site associated with the team.
    */
   teamSitePath?: string;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.channel.ownerTeamTenantId} instead
+   *
    * The tenant ID of the host team.
    */
   hostTeamTenantId?: string;
 
   /**
-   * @hidden
+   * @deprecated
+   * As of 2.0.0, please use {@link app.Context.channel.ownerTeamGroupId} instead
```

Comment:
This looks like it's actually app.Context.channel.ownerGroupId (not ownerTeamGroupId).